packages/swarm-docs/src/pages/Button.mdx

```diff
@@ -23,9 +22,56 @@ Buttons are the primary way to interact with content.
 ## Props
 <PropsTable propMetaData={Button.__docgenInfo} />
 
+## Anatomy
+All button styles and sizes share the same anatomy.
+<img src="/static/docs-images/button-anatomy.png" width="810" height="252" />
 
-## a11y
+## Styles
+
+** Default **
+
+<img src="/static/docs-images/button-secondary-states.png" width="810" height="132" />
+<br/>
+
+** Primary **
+
+<img src="/static/docs-images/button-primary-states.png" width="810" height="132" />
+<br/>
+
+** Neutral **
+
+<img src="/static/docs-images/button-neutral-states.png" width="810" height="132" />
+<br/>
+
+** Bordered **
+
+<img src="/static/docs-images/button-bordered-states.png" width="810" height="132" />
+<br/>
+
+** Disabled **
 
+<img src="/static/docs-images/button-disabled-states.png" width="250" height="132" />
+
+
+
+
+## Sizing
+
+** Size & Height **
+
+Any button can be configured to one of three distinct sizes. Size dictates label style and top/bottom padding. Button padding uses spacing variables which are dynamic based on viewport.
+<br/>
+<img src="/static/docs-images/button-sizing-size.png" width="600" height="373" />
+<br/><br/>
```

Comment:
should we add some CSS for `img` to have a bottom margin in the docs to avoid the need for this `<br />` noise?

Reply:
It's a good point. Maybe this is something we can circle back to once I build out the layouts for other component pages. Ideally we should try to eliminate the `<br>`s. But I want to make sure I'm not baking in unnecessary margins into these elements.

Reply:
Feel free to remove all `<br/>` tags and replace with CSS. I assumed that would happen at some point; I just never knew what the plan/priority for styling the docs would be. So, I added the tags in as a quick fix doc pages so I could have pages that are formatted and legible.